charts/fluentd/Chart.yaml

```
@@ -4,7 +4,7 @@ description: A Helm chart for shipping Kubernetes logs via Fluentd.
 keywords:
   - logging
   - fluentd
-version: 0.1.0
+version: 0.1.1
```

Since we're adding a new ability, let's bump to 0.2.0
```suggestion
version: 0.2.0
```